modules/gitops-argocd-cli-logging-in-to-argocd-server.adoc

```diff
@@ -0,0 +1,76 @@
+// Module is included in the following assemblies:
+//
+// * gitops_cli_argocd/logging-in-to-argocd-server.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="gitops-argocd-cli-logging-in-to-argocd-server{context}"]
+= Logging in to the Argo CD server
+
+After you install and configure the {gitops-shortname} `argocd` CLI, to execute commands in the default mode, you must log in to the Argo CD server.
+
+.Prerequisites
+
+* You must have the {gitops-shortname} `argocd` CLI tool installed and configured.
+
+.Procedure
+
+Use your credentials to log in and remain logged in throughout the session. 
+
+[TIP]
+====
+ If the login session times out, you can use the `relogin` command to log in again. When done using the `argocd` commands, you can log out using the `logout` command.
+====
+
+. Get the admin account password for the Argo CD server:
++
+[source,terminal]
+----
+$ ADMIN_PASSWD=$(oc get secret openshift-gitops-cluster -n openshift-gitops -o jsonpath='{.data.admin\.password}' | base64 -d)
+----
+
+. Get the Argo CD server URL:
++
+[source,terminal]
+----
+$ SERVER_URL=$(oc get routes openshift-gitops-server -n openshift-gitops -o jsonpath='{.status.ingress[0].host}')
+----
+
+. Log in to the Argo CD server.
++
+.. Display the line of text or string of the admin account password:
++
+[source,terminal]
+----
+$ echo ${ADMIN_PASSWD}
+----
+
+.. Use the admin account password by enclosing it in single quotes:
++
+[IMPORTANT]
+====
+Passwords can contain special characters like `$`, which would be interpreted as shell variables. This can cause the command to fail because as interpreted by shell, a wrong value of password would be sent to the server. Always use single quotes to enclose the actual value of the password to avoid such errors.
+====
++
+[source,terminal]
+----
+$ argocd login --username admin --password ${ADMIN_PASSWD} ${SERVER_URL}
+----
++
+.Example
+[source,terminal]
+----
+$ argocd login --username admin --password '<password>' openshift-gitops.openshift-gitops.apps-crc.testing
+----
+
+. Execute the `argocd` commands:
```

Comment:
Is this step necessary to document? If we're describing how to log in, that has already been described. I think this step can be removed, personally.

Reply:
@anandf WDYT?

Reply:
removing this.